plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveGlueMetadataListing.java

```diff
@@ -49,6 +52,8 @@ public class TestHiveGlueMetadataListing
 {
     public static final String FAILING_TABLE_WITH_NULL_STORAGE_DESCRIPTOR_NAME = "failing_table_with_null_storage_descriptor";
     public static final String FAILING_TABLE_WITH_NULL_TYPE = "failing_table_with_null_type";
+    public static final String FAILING_TABLE_WITH_BAD_HIVE_TYPE = "failing_table_with_bad_hive_type";
+    public static final String FAILING_TABLE_WITH_NULL_SERDER = "failing_table_with_null_serder";
```

Comment:
```suggestion
    public static final String FAILING_TABLE_WITH_NULL_SERDE = "failing_table_with_null_serde";
```

Reply:
Good catch. I've updated this in the force push. And checked for this typo that I had made all over the place.
```
% git grep -i "serder" | wc -l
       0
```